docs/data/toolpad/concepts/deployment.md

```diff
@@ -2,24 +2,78 @@
 
 <p class="description">Deploying a Toolpad app is like deploying any other React/Node application.</p>
 
-## Build steps
+## Pre-requisites
+
+- This guide assumes that you have copied your project folder to the server, and all subsequent commands are run inside it.
+
+- Make sure to add the following scripts to your `package.json`
+
+```json
+ "scripts": {
+   "dev": "toolpad dev"
+   "build": "toolpad build"
+   "start": "toolpad start"
+ }
+```
+
+## Install step
+
+Install required depdencies via:
+
+<codeblock storageKey="toolpad-package-manager">
```

Comment:
to keep in line with https://github.com/mui/mui-toolpad/blob/5a1a5717e8aee85a428e55bfb18727309c60da89/docs/data/toolpad/getting-started/installation.md?plain=1#L10 perhaps we could use `package-manager` everywhere?

```suggestion
<codeblock storageKey="package-manager">
```

Or replace the installation instructions with `toolpad-package-manager`?

Reply:
I only added the `toolpad` prefix to prevent a clash with other core docs, but on second thought it should probably work across all docs pages? We can keep `package-manager` 👍